Rakefile

```diff
@@ -149,8 +149,8 @@ namespace :ciinabox do
     dns_domain = config['dns_domain']
     script = "
     openssl req -nodes -new -x509 -newkey rsa:4096 -days 3650 \
-      -keyout #{ciinaboxes_dir}/#{ciinabox_name}/ssl/ciinabox.key \
-      -out #{ciinaboxes_dir}/#{ciinabox_name}/ssl/ciinabox.crt \
+      -keyout #{ciinaboxes_dir}/#{ciinabox_name}/ssl/#{ciinabox_name}_ciinabox.key \
```

Comment:
I would define the ciinabox_name variable to include the _ciinabox part if the ciinabox has a name otherwise it should have the default value of ciinabox. This would make it backward compatible with existing ciinaboxes that don't have a name